crates/bevy_core/src/lib.rs

```diff
@@ -50,6 +50,7 @@ impl Plugin for CorePlugin {
 
         app.init_resource::<Time>()
             .init_resource::<FixedTimesteps>()
+            .init_resource::<SystemRegistry>()
```

Comment:
This resource is required by bevy_ecs itself. I think it should either be registered directly in `World::new` or stored as it's own field of `World`.

Reply:
This sort of thing is why I want #3786. Sounds good to me; I think the former solution is less messy.